persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_24/generated/rainier/generated_client.bal

```diff
@@ -112,9 +112,9 @@ public client class RainierClient {
     }
 
     isolated resource function delete building/[string buildingCode]() returns Building|persist:Error {
-        Building result = check self->/building/[buildingCode].get();
+        Building 'object = check self->/building/[buildingCode].get();
```

Comment:
why we change this variable name?

Reply:
this change is from a disabled test case, This can be removed as in the latest design, we only checks schema file for the errors. this file will be updated in [145](https://github.com/ballerina-platform/persist-tools/pull/145)